prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetAnalysisTask.java

```diff
@@ -108,13 +111,14 @@ else if ("Failed".equals(success.value().state())) {
 		throw new IllegalArgumentException();
 	}
 
-	private void sendEmail(String reportURL, String recipient) {
-		if (recipient == null) {
-			return;
-		}
-		messageConsumer.accept(new Message(recipient, "parodos-task-notificaion+mailtrap@redhat.com",
-				"Parodos: Analysis report is done",
-				String.format("The analysis report is done. Find it here %s", reportURL)));
+	private void sendNotification(String reportURL) {
+		var request = new NotificationMessageCreateRequestDTO();
+		request.setMessageType("text");
+		request.setBody("Report URL: " + reportURL);
+		request.subject("Migration Analysis Report Completed");
+		request.setUsernames(List.of("test"));
```

Comment:
Shouldn't this be the user that invoked the workflow instead of the default "test" value?
can be obtained by `SecurityUtils.getUsername()`